server/models/Prospect.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+
+var Status = Object.freeze({
+    OPEN : 'open',
+    CLOSED: 'closed'
+});
+
+/**
+ * 
+ * We will want to make the 'owned_by' field a required field once we have workable user_id's
+ * 
+ **/
+
+const ProspectSchema = new Schema({
+    first_name: { type: String, required: true},
+    last_name: { type: String, required: true},
+    email: { type: String, required: true, index: true},
+    owned_by: {type: mongoose.Schema.Types.ObjectId, ref: 'User', required : false},
```

Comment:
I think I wanted to index owned_by as opposed to email actually - we are going to be doing a lot of fetching all prospects by owned_by as opposed to email

Also, i mentioned this in Darren's PR too, we want to use camelCase for the fields names - see https://intellipaat.com/community/28608/what-are-naming-conventions-for-mongodb